packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -109,3 +144,46 @@ impl Modify for MergeAddon {
         );
     }
 }
+
+/// Addon to allow external references in schemas.
+///
+/// Any component that starts with `EXTERNAL_` will transform into a relative URL in the schema and
+/// receive a `.json` ending.
+///
+/// For example the `EXTERNAL_Entity` component will be transformed into `./models/Entity.json`
+struct ExternalRefAddon;
+
+impl Modify for ExternalRefAddon {
+    fn modify(&self, openapi: &mut openapi::OpenApi) {
+        for path_item in openapi.paths.paths.values_mut() {
+            for operation in path_item.operations.values_mut() {
+                if let Some(request_body) = &mut operation.request_body {
+                    modify_component_references(&mut request_body.content);
+                }
+
+                operation
+                    .responses
+                    .responses
```

Comment:
🙃

Reply:
Also loving the swap to an `iter` function to avoid even more nesting 😁

Reply:
I rewrote these to `for in` loops but missed this one. My bad :)

Hopefully better in d19e8ae02dbb27ca4e1533c58269cdfe5466a016

Reply:
Better :) Although my 🙃 was about the `responses.responses` but we're not in control of that haha